### DISCUSSION TITLE
Form Page

### DISCUSSION THREAD

**Discussion by havvanur160401046** [serenity-is/Serenity#5459] [Help]
![image](https://user-images.githubusercontent.com/24252615/107341279-d341a000-6acf-11eb-9f8a-61edaa04a47a.png)
![image](https://user-images.githubusercontent.com/24252615/107341340-e3597f80-6acf-11eb-9f30-05a060f3cf3b.png)
my form page is like the second image
but I want it to be like the first relative, what should I do

**Comment by arunkumar12712** [comment]
@Serenity.Decorators.panel()  add in dialog

**Reply by havvanur160401046** [reply]
Thank you :)

**Reply by minhhungit** [reply]
@havvanur160401046 please mark the answer as answered if it helped
